napari/utils/events/event.py

```diff
@@ -193,7 +193,9 @@ def __repr__(self) -> str:
                 attr = getattr(self, name)
 
                 attrs.append(f"{name}={attr!r}")
-            return "<{} {}>".format(self.__class__.__name__, " ".join(attrs))
+            return (  # noqa TRY301
```

Comment:
`TRY301` is about [direct raises in try blocks](https://github.com/guilatrova/tryceratops/blob/main/docs/violations/TC301.md), is it somehow misfiring here?